site/app/libraries/database/DatabaseQueriesPostgresql.php

```diff
@@ -190,7 +190,7 @@ public function getGradeables($g_ids = null, $user_ids = null, $section_key="reg
                 return $return;
             }
         }
-        $keys = array("registration_section", "rotating_section");
+        $keys = array("registration_section", "rotating_section", "registration_section_by_first", "registration_section_by_last");
```

Comment:
I'd prefer you add a new argument to the function `$sort_key` or something and not overload `$section_key` since it'll lead to a lot of duplicate stuff as we apply the same sorts to both registration and rotating sections (and make it harder to add new sorting methods in the future).